projects/plugins/jetpack/extensions/blocks/blogroll/blogroll-item/blogroll-item.php

```diff
@@ -74,7 +74,7 @@ function load_assets( $attr, $content, $block ) {
16' height='985' stroke='%23CCCCCC'/%3E%3C/g%3E%3C/g%3E%3C/g%3E%3Cpath d='M-5744 -1274H2020V-1276H-5744V-1274ZM2021 -1273V635H2023V-1273H2021ZM2020 636H-5744V638H2020V636ZM-5745 635V-1273H-5747V635H-5745ZM-5744 636C-5744.55 636 -5745 635.552 -5745 635H-5747C-5747 636.657 -5745.66 638 -5744 638V636ZM2021 635C2021 635.552 2020.55 636 2020 636V638C2021.66 638 2023 636.657 2023 635H2021ZM2020 -1274C2020.55 -1274 2021 -1273.55 2021 -1273H2023C2023 -1274.66 2021.66 -1276 2020 -1276V-1274ZM-5744 -1276C-5745.66 -1276 -5747 -1274.66 -5747 -1273H-5745C-5745 -1273.55 -5744.55 -1274 -5744 -1274V-1276Z' fill='black' fill-opacity='0.1'/%3E%3C/g%3E%3Cdefs%3E%3Cpattern id='pattern0' patternContentUnits='objectBoundingBox' width='1' height='1'%3E%3Cuse xlink:href='%23image0_37_971' transform='scale(0.0125)'/%3E%3C/pattern%3E%3CclipPath id='clip0_37_971'%3E%3Crect width='1595' height='1068' fill='white' transform='translate(-234 -815)'/%3E%3C/clipPath%3E%3CclipPath id='clip1_37_971'%3E%3Crect x='-22' y='-111' width='350' height='298' fill='white'/%3E%3C/clipPath%3E%3Cimage id='image0_37_971' data-name='no-site-icon.png' width='80' height='80' xlink:href='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAABQCAYAAACOEfKtAAAACXBIWXMAABYlAAAWJQFJUiTwAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAATUSURBVHgB7ZzvbxJpEMenBwEKJ1ybthaFi+f1ohd9cblLLtF/X6OJJr6y0cYfbWwV1AYFBQtpo893Gxr26S67MDNLY+aTNAJdof0wz/PMzPPQpXv3H34nY25+IYOFCWRiApmYQCYmkIkJZGICmZhAJiaQiQlkYgKZmEAmJpCJCWRiApmYQCYmkIkJZGICmeTpglCtVqlSXqZazf1bKVMul6N8/vTHOz4+ptFoREdHQ+r3B9TtfaFer0cXgYUKzOdzVK9vUn3z8pms6OvywVe5XKbV1RVquseGw2Egcn//wN0e0aJYiECIazauBvLmpVgs0sY6vtbow8fDhYnMXGC9fjmQNy3iZgUSa9VLTuLbQGaWZCrwj2u/x0Yd5rnOp8/U6/boq5vnMOfhMTAevqViIRjCVSfLfwMQkVtb16ns5s+9vTeUFUtZbKxjyN688VewUMTx6PGTM2FpuHvn/9jv9QcD2t5+5p7vhLTJJI25devvqfIAhrYUFRetN9wblgXqAjFs8QslgXmx2bxKUtTcG3bNvbY2qnMgJvdZVtpAYiMs8cHDR+evSyn6inttrMytVpu0UIvAopvwG4IRNQbyfMmT+PPo6YqfIy3UBDYbDbdqFkmaafKQxjzdfh6SCHmNhvwbOUZFIKJvY2ONsgTy9g/e0sCtwK3W+9D3MJS1olBFIKJPAkhJex3knd13t49cqTdJfXP+qmcaKgKrtUvExZcy63Wtd+GFg1M2TkNcIPI97tw3Td5kVE677uPh4bm5MCkXnQfxNAY1KYekyPOHahyoQjqdz6G5GG0y6TaYuMBKpULzknrYxlyTlOIguZbOCcWHcKlUoHlIKy+OJHmgUlkmacQjsFBIJzCqwpiXNPJALidfeIlHoGSfLw1p5QGNXFC9mZA2l5uXpmKVkQZxgX4t2mq31SWmRaM/KD7e8ENODuOCywmxOCQtEHEN0uD/Cr0BQ686kUA8AgeDb6H7v6boBU5Dsk+osekkLrDb7YbuV2v87D+pA5OWfr9P0ogL7HsRuLqyotqPm2WIYx9ZGvE5EKUSFpLxPAh56+vr7ArAnyMn88ioOXbrz+uhMg7zn8ZpBpU05nw/Tm7DKC1+R0gj+oCOwHY42rBnm9ROkkx1sOj4HSGtVEpFIFIZf8hiIUCnOg6pdAWv4S86OK2gkcIAtUoEQvx+3G23PywhMe4aPDf2oCeBOM1EXk0gonD/4F3oMQxlrsS4rs1YXtTQ1Yo+oFoLYxj7QzmtxMjHY+SVy8uR8vDa2oeN1JsJu3tv3AoYTh8g8b9//5mpwoiTh+e4HSEP52N2MzhklEnvaWfnRRAh/hEPTPY4vZB0LM2XF+SWa2tUv7IZuf+CfO/5zkvKgqUs/+xJ0vG2nsvVOp1Pbkty5KLqZvA45CEtQlMCdTVKw9WV32L7jhi2uz/b8bZJEHGNiDyNC94ALFqa52CiyPyEKoYqqoJg+AqdXoA0yJvlfKEUS4v8y0VYTCASZdesEQlZKBlb7fcLETdmoaf0kZ+9fPU6uI1Nb+wpY++2VCoGm1OTH3M4OTk5/YhDtxd0fOxjDh4QclGkzIJ9UomJCWRiApmYQCYmkIkJZGICmZhAJiaQiQlkYgKZmEAmJpCJCWRiApmYQCYmkIkJZGICmfwAzeIV04TFHZIAAAAASUVORK5CYII='/%3E%3C/defs%3E%3C/svg%3E%0A";
```

Comment:
We can add a file and use `plugin_url` to get the path. Base64 is ~67% as efficient as binary files. And it's noisy.